packages/react-performance/src/context.ts

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {Performance} from '@shopify/performance';
 
-export const PerformanceContext = React.createContext<Performance | null>(
-  typeof window === 'undefined' ? null : new Performance(),
```

Comment:
cc @vsumner another place I kept going back and forth about using null or undefined.